cime_config/testdefs/testmods_dirs/clm/collapse_pfts_16_to_16/user_nl_clm

```diff
@@ -0,0 +1 @@
+for_testing_allow_non_annual_changes = .true.
```

Comment:
I wouldn't think this user_nl_clm is needed for this test. Can this test simply use the 'default' testmod rather than having its own testmod?

Reply:
Removing.
(This was my original thought process: I did not know whether it helped with "testing" in general and assumed that it would not likely hurt anything to leave there.)